src/companies/models/point.py

```diff
@@ -19,6 +19,6 @@ def __str__(self) -> str:
         return f"{self.company.name} - {self.address}"
 
     class Meta:
-        ordering = ("id",)
+        ordering = ("company",)
```

Comment:
I would suggest ordering by address. It would be better as we actually always query points for specific company.